cmd/dmt/root.go

```diff
@@ -46,13 +46,17 @@ func execute() {
 		CompletionOptions: cobra.CompletionOptions{
 			HiddenDefaultCmd: true,
 		},
+		Run: func(cmd *cobra.Command, _ []string) {
+			_ = cmd.Help()
+		},
```

Comment:
Move it under PersistentPreRun and fix error handling
```suggestion
		RunE: func(cmd *cobra.Command, args []string) error {
			return cmd.Help()
		},
```